config-templates/aws/aws-efs-csi-driver/2.0.3/metadata.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "2.0.3",
+    "revision": "1",
+    "enabled": "true",
+    "status": "supported",
+    "description": "AWS EFS CSI Driver version 2.0.3",
+    "tags": [
+      {
+        "name": "provider",
+        "value": "AWS"
+      },
+      {
+        "name": "offering-name",
+        "value": "aws-efs-csi-driver"
+      },
+      {
+        "name": "offering-type",
+        "value": "non-sds"
+      },
+      {
+        "name": "provider-url",
+        "value": "https://console.aws.amazon.com/support/home#/"
+      },
+      {
+        "name": "support-msg",
+        "value": "For support contact AWS"
+      },
+      {
+        "name": "provider-logo",
+        "value": "logo/aws-efs-csi-driver.svg"
+      },
+      {
+        "name": "supported-envs",
+        "value": "AWS"
+      },
+      {
+        "name": "supported-ocp-versions",
+        "value": "4.14, 4.15"
```

Comment:
<img width="945" alt="Screenshot 2024-06-07 at 10 13 04 AM" src="https://github.com/IBM/ibm-satellite-storage/assets/34068620/57062575-64ac-41ae-a116-fe308a181924">

As per this, I think it will work with all supported OCP versions in IBMCloud. Hence lets include 4.12, 4.13, 4.14 & 4.15